tests/backends/ros/messages/test_std_msgs.py

```diff
@@ -18,3 +22,20 @@ def test_nested_repr():
     t = Time(80, 20)
     h = Header(seq=10, stamp=t, frame_id='/wow')
     assert repr(h) == "Header(seq=10, stamp=Time(secs=80, nsecs=20), frame_id='/wow')"
+
+
+def test_float32_multiarray():
+    m = Float32MultiArray(data=[1.3, 0.5, 3.4])
+    assert(repr(m) == "Float32MultiArray(layout=MultiArrayLayout(dim=[], data_offset=0), data=[1.3, 0.5, 3.4])")
```

Comment:
Ironpython seems to have some rounding issues: 
<img width="927" alt="Screenshot 2021-07-26 at 13 52 40" src="https://user-images.githubusercontent.com/57254617/126984875-fe225e24-0dc5-4289-b3f8-6c844297b9dd.png">
A quick and dirty way around this would be to change 3.4 to 3.0.

Reply:
:+1: